From e9fbaa1de69cb71301c6ecd05575636ffcfa53e0 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Mon, 20 Aug 2018 00:14:24 +0100
Subject: [PATCH 220/249] manifest: Stop creating Android.bp symlinks

 * As per the content of the root Android.bp file:

   // Soong finds all Android.bp and Blueprints files in the source tree,
   // subdirs= and optional_subdirs= are obsolete and this file no longer
   // needs a list of the top level directories that may contain Android.bp
   // files.

Change-Id: I9185e66f8ac1bad1518e8cf0342f3972d1905b56
---
 default.xml          |  5 +----
 snippets/lineage.xml | 12 +++---------
 2 files changed, 4 insertions(+), 13 deletions(-)

diff --git a/default.xml b/default.xml
index 454b2ff..c122bca 100644
--- a/default.xml
+++ b/default.xml
@@ -32,7 +32,6 @@
   <project path="build/blueprint" name="platform/build/blueprint" groups="pdk,tradefed" remote="aosp" />
   <project path="build/kati" name="LineageOS/android_build_kati" groups="pdk,tradefed" />
   <project path="build/soong" name="LineageOS/android_build_soong" groups="pdk,tradefed" >
-    <linkfile src="root.bp" dest="Android.bp" />
     <linkfile src="bootstrap.bash" dest="bootstrap.bash" />
   </project>
   <project path="art" name="LineageOS/android_art" groups="pdk" />
@@ -462,9 +461,7 @@
   <project path="hardware/qcom/gps" name="LineageOS/android_hardware_qcom_gps" groups="qcom,qcom_gps,pdk-qcom" />
   <project path="hardware/qcom/keymaster" name="LineageOS/android_hardware_qcom_keymaster" groups="qcom,qcom_keymaster,pdk-qcom" />
   <project path="hardware/qcom/media" name="LineageOS/android_hardware_qcom_media" groups="qcom,pdk-qcom" />
-  <project path="hardware/qcom/neuralnetworks/hvxservice" name="platform/hardware/qcom/neuralnetworks/hvxservice" groups="wahoo" remote="aosp" >
-    <copyfile dest="hardware/qcom/Android.bp" src="Android.bp"/>
-  </project>
+  <project path="hardware/qcom/neuralnetworks/hvxservice" name="platform/hardware/qcom/neuralnetworks/hvxservice" groups="wahoo" remote="aosp" />
   <project path="hardware/qcom/wlan" name="LineageOS/android_hardware_qcom_wlan" groups="qcom_wlan,pdk-qcom" />
   <project path="hardware/ril" name="LineageOS/android_hardware_ril" groups="pdk" />
   <project path="kernel/configs" name="kernel/configs" groups="vts,pdk" remote="aosp" />
diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 67c2884..4c14048 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -21,9 +21,7 @@
   <!--<project path="external/unrar" name="LineageOS/android_external_unrar" />-->
   <!--<project path="external/vim" name="LineageOS/android_external_vim" />-->
   <!--<project path="external/zip" name="LineageOS/android_external_zip" />-->
-  <!--<project path="hardware/lineage/interfaces" name="LineageOS/android_hardware_lineage_interfaces" >-->
-  <!--  <copyfile dest="hardware/lineage/Android.bp" src="Android.bp"/>-->
-  <!--</project>-->
+  <!--<project path="hardware/lineage/interfaces" name="LineageOS/android_hardware_lineage_interfaces" />-->
   <!--<project path="hardware/lineage/lineagehw" name="LineageOS/android_hardware_lineage_lineagehw" />-->
   <!--<project path="hardware/lineage/telephony" name="LineageOS/android_hardware_lineage_telephony" />-->
   <!--<project path="hardware/ti/omap4" name="LineageOS/android_hardware_ti_omap4" />-->
@@ -101,17 +99,13 @@
   <!--<project path="system/qcom" name="LineageOS/android_system_qcom" groups="qcom" />-->
   <project path="system/tools/dtbtool" name="LineageOS/android_system_tools_dtbtool" />
   <project path="vendor/codeaurora/telephony" name="LineageOS/android_vendor_codeaurora_telephony" />
-  <!--<project path="vendor/nxp/interfaces/opensource/nfc" name="LineageOS/android_vendor_nxp_interfaces_opensource_nfc" >-->
-  <!--  <linkfile dest="vendor/nxp/interfaces/Android.bp" src="prop_pickup.bp"/>-->
-  <!--</project>-->
+  <!--<project path="vendor/nxp/interfaces/opensource/nfc" name="LineageOS/android_vendor_nxp_interfaces_opensource_nfc" />-->
   <!--<project path="vendor/nxp/opensource/external/libnfc-nci" name="LineageOS/android_vendor_nxp_opensource_external_libnfc-nci" />-->
   <!--<project path="vendor/nxp/opensource/frameworks" name="LineageOS/android_vendor_nxp_opensource_frameworks" />-->
   <!--<project path="vendor/nxp/opensource/packages/apps/Nfc" name="LineageOS/android_vendor_nxp_opensource_packages_apps_Nfc" />-->
   <!--<project path="vendor/qcom/opensource/cryptfs_hw" name="LineageOS/android_vendor_qcom_opensource_cryptfs_hw" />-->
   <!--<project path="vendor/qcom/opensource/dataservices" name="LineageOS/android_vendor_qcom_opensource_dataservices" />-->
-  <project path="vendor/qcom/opensource/interfaces" name="LineageOS/android_vendor_qcom_opensource_interfaces" >
-    <copyfile dest="vendor/qcom/opensource/Android.bp" src="os_pickup.bp"/>
-  </project>
+  <project path="vendor/qcom/opensource/interfaces" name="LineageOS/android_vendor_qcom_opensource_interfaces" />
   <!--<project path="vendor/qcom/opensource/rcs-service" name="LineageOS/android_vendor_qcom_opensource_rcs-service" />-->
 
   <!-- Infrastructure -->
-- 
2.17.1

